Makefile

```diff
@@ -32,6 +32,7 @@ test:
 	$(DOCKER_COMPOSE) up -d db
 	./mysql/bin/wait_for_mysql
 	$(DOCKER_COMPOSE) run -e RACK_ENV=test --rm app ./bin/rails db:create db:schema:load
+	$(DOCKER_COMPOSE) run -e RAILS_ENV=test --rm app ./bin/rails db:migrate
```

Comment:
Small one but is this a line on its own on purpose?

Could just append it at the end of this command like so:
$(DOCKER_COMPOSE) run -e RACK_ENV=test --rm app ./bin/rails db:create db:schema:load db:migrate